crates/primitives/src/chain/spec.rs

```diff
@@ -603,6 +604,12 @@ pub enum ForkCondition {
     Never,
 }
 
+impl Display for ForkCondition {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{self:?}")
```

Comment:
This displays things like

```
TTD { fork_block: None, total_difficulty: 0x000000000000000000000000000000000000000000000c70d808a128d7380000_U256 }
```

which is not easy to read.

ptal at `geth`'s default output which we should use as a reference here